substrate/frame/treasury/src/migration.rs

```diff
@@ -0,0 +1,74 @@
+use super::*;
+use alloc::collections::BTreeSet;
+use core::marker::PhantomData;
+use frame_support::traits::UncheckedOnRuntimeUpgrade;
+
+/// The log target for this pallet.
+const LOG_TARGET: &str = "runtime::treasury";
+
+mod v1 {
```

Comment:
we need some name like `cleanup_proposals` and doc (proposals creation via referenda were deprecated and removed and a client can do with this migration).

Reply:
`v1` usually refers to the storage schema version

Reply:
Removed, also named it CleanupProposals